testservice/src/main/kotlin/com/wire/kalium/testservice/models/BreakSessionRequest.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.testservice.models
+
+data class BreakSessionRequest(
+    val clientId: String = "",
+    val userId: String = "",
+    val userDomain: String = "staging.zinfra.io",
```

Comment:
Why assign default values?

Reply:
Do you mean on userDomain or on the other values? The idea in the case of userDomain is that you don't have to enter it every time in the swagger ui when using the endpoint manually (which we mostly use against staging).

Reply:
Ah ok. I see know that this is for the test service API.